.github/workflows/build.yml

```diff
@@ -80,3 +80,31 @@ jobs:
 
     - name: Run-CC-Samples-with-LOC-wo-L3_LOC_ENABLED
       run: BUILD_MODE=${{ matrix.build_type }} CC=g++ CXX=g++ LD=g++ make run-cc-tests
+
+    # -------------------------------------------------------------------------
+    - name: Build-and-Run-LOC-unit-test
+      run: |
+        BUILD_MODE=${{ matrix.build_type }} make clean
+        BUILD_MODE=${{ matrix.build_type }} CC=g++ CXX=g++ LD=g++ BUILD_VERBOSE=1 L3_LOC_ENABLED=2 make run-loc-tests
```

Comment:
build-and-test simple unit-test to cross-check return type of `__LOC__` encoding macro when LOC-ELF is enabled.